amida-tech/indaba-client
# Feature/inba 661 survey tooltips

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
Adds a toolip to the required question, add link and attach file text in the survey question panel

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-661
#### How should this be manually tested?
Go to any project that has questions added to the survey. Hover over each one. Ensure the tooltip matches the project managers (Kate) comment on the ticket.
#### Background/Context
No
#### Screenshots (if appropriate):
